### PR TITLE
Performance

### PR DESCRIPTION
**Pull request by Meberem**
I am only bundling that which is neccessary now. Fully optimized the bundle is 8.39KB and I have attached the Chrome Lightout audit for your review. It looks like the few remaining optimizations are around css and image compression/omptimization.

image

### CODE REVIEW

**Comment by bzaar** [comment]
Thank you! What does the same report look like for the current master version?

**Comment by Meberem** [comment]
image

**Comment by bzaar** [comment]
That’s gramdict.ru vs localhost. Can we see a comparison on the same domain?

**Comment by Meberem** [comment]
I have updated gramdict.ru with the site, please see the report below
image

File: gramdictru/package.json [hunk 1]
```diff
@@ -10,12 +10,12 @@
   "dependencies": {
     "@types/react": "^16.7.20",
     "@types/react-dom": "^16.0.11",
-    "@types/react-infinite-scroll-component": "^4.4.0",
     "@types/styled-components": "^4.1.6",
     "axios": "^0.18.0",
     "csv-string": "^3.1.5",
```
Comment (bzaar) [inline gramdictru/package.json]: I thought we're not using csv-string anymore?

Comment (Meberem) [reply gramdictru/package.json]: apparently my first uninstall command didn't take... I have pushed up the changes

**Pull request merged**